blog/content/post/2024-12-13-ram-data-decay-research.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Research of RAM data remanence times
+abstract: "Dynamic RAM must be periodically refreshed to maintain its content.
+          JEDEC specifies how often this has to be done to ensure proper
+          operation. In this post, we will see for how long the data can be
+          recovered if it isn't."
+cover: /covers/image-file.png
```

Comment:
Needs a proper cover

Reply:
https://github.com/3mdeb/news-and-ideas/pull/621/commits/478ab8793dac42e71f71d1b2f30aeb7b92d053c6